go.mod

```diff
@@ -44,7 +44,6 @@ require (
 	github.com/golang/mock v1.4.3
 	github.com/golang/protobuf v1.3.4
```

Comment:
should we add a comment here to maintain the old version?

Reply:
do you mean add comment in order to keep the geth's version?
I prefer to prepare another PR to solve the remain TODOs

Reply:
@baptiste-b-pegasys can you create an issue to track this, so we are aware we need to put back the original version from geth?

Reply:
in the meantime, I can merge this.